src/aeb_fate_data.erl

```diff
@@ -208,17 +208,15 @@ ordinal(T) when ?IS_FATE_BOOLEAN(T)   -> 1;
 ordinal(T) when ?IS_FATE_ADDRESS(T)   -> 2;
 ordinal(T) when ?IS_FATE_CHANNEL(T)   -> 3;
 ordinal(T) when ?IS_FATE_CONTRACT(T)  -> 4;
-%%                                       5
-ordinal(T) when ?IS_FATE_ORACLE(T)    -> 6;
+ordinal(T) when ?IS_FATE_ORACLE(T)    -> 5;
+ordinal(T) when ?IS_FATE_ORACLE_Q(T)  -> 6;
 ordinal(T) when ?IS_FATE_BYTES(T)     -> 7;
-%%                                       8;
-ordinal(T) when ?IS_FATE_BITS(T)      -> 9;
-ordinal(T) when ?IS_FATE_STRING(T)    -> 10;
-ordinal(T) when ?IS_FATE_TUPLE(T)     -> 11;
-ordinal(T) when ?IS_FATE_MAP(T)       -> 12;
-ordinal(T) when ?IS_FATE_LIST(T)      -> 13;
-ordinal(T) when ?IS_FATE_VARIANT(T)   -> 14;
-ordinal(T) when ?IS_FATE_ORACLE_Q(T)  -> 15.
+ordinal(T) when ?IS_FATE_BITS(T)      -> 8;
+ordinal(T) when ?IS_FATE_STRING(T)    -> 9;
+ordinal(T) when ?IS_FATE_TUPLE(T)     -> 10;
+ordinal(T) when ?IS_FATE_MAP(T)       -> 11;
+ordinal(T) when ?IS_FATE_LIST(T)      -> 12;
+ordinal(T) when ?IS_FATE_VARIANT(T)   -> 13.
```

Comment:
The reason to not renumber these is that the numbers are also used in `lt` below.

Reply:
If the FATE_LIST_VALUE match is replaced by IS_FATE_LIST then the number could be removed from lt.

Reply:
And the FATE_STRING.

Reply:
This is only tested in the quick check tests. Perhaps leave the renaming and I can do it in a separate PR also adding tests. https://www.pivotaltracker.com/story/show/166975637